ExampleMod/Content/Projectiles/ExampleSpearProjectile.cs

```diff
@@ -0,0 +1,94 @@
+﻿using ExampleMod.Content.Dusts;
+using Microsoft.Xna.Framework;
+using Terraria;
+using Terraria.ModLoader;
+using static Terraria.ModLoader.ModContent;
+
+namespace ExampleMod.Content.Projectiles
+{
+	public class ExampleSpearProjectile : ModProjectile
+	{
+		public override void SetStaticDefaults() {
+			DisplayName.SetDefault("Spear");
+		}
+
+		public override void SetDefaults() {
+			projectile.width = 18; // The width of the projectile's hitbox in pixels.
+			projectile.height = 18; // The height of the projectile's hitbox in pixels.
+			projectile.aiStyle = 19; // Selects which vanilla code to use for the AI method. Modders can use vanilla aiStyle to mimic AI code already in the game.
```

Comment:
Comment should just say what 19 represents. What AI is this mimicking (spears presumably)

Reply:
Why do you have an AIStyle if you override AI below?

Reply:
Surprisingly, if I don't use Alstyle the projectile has a offset issue so I included it to fix it.